web/templates/shared/_info.svg.eex

```diff
@@ -0,0 +1,4 @@
+<svg viewBox="0 0 200 200" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
If you want, you can get rid of the `xmlns:xlink="http://...`, but I'd leave the first name space.